### DISCUSSION TITLE
[PROPOSAL] Move feature requests to GitHub issues

### DISCUSSION THREAD

**Discussion by MilanKovacic** [nrwl/nx#23131] [Feature Requests]
Using GitHub Discussions to propose and track feature requests currently seems less effective compared to GitHub Issues, given the comparatively lower level of activity in Discussions. Most open source projects conventionally use GitHub Issues for feature requests, which tends to engage more community interaction and provides better visibility and tracking.

The suggestion is to shift feature requests back to GitHub Issues. To manage these efficiently, adopting an automated process similar to the one used by the Visual Studio Code team could be beneficial. The proposed process is as follows:

"When a feature request is submitted, it becomes a candidate for our backlog. The community has 60 days to upvote the issue. If it receives 20 upvotes within this period, we will move it to our backlog. If not, the issue will be closed. More details on handling feature requests can be found in our documentation."

This approach not only aligns with common practices across open source projects but also empowers the community to actively participate in prioritizing the features that are most important to them.

**Comment by jaysoo** [comment]
@MilanKovacic Let's keep the feature requests here, and we'll make sure discussions are attended to more frequently.

The goal of moving feature requests to discussions is to reduce noise in our GitHub issues. It takes time to triage them, and having feature requests mixed in with bugs adds to the time before we get to high-priority issues.

Another concern is that, for better or worse, one metric for the quality of OSS projects is the number of open issues, how fast they are closed, etc. 

Let's try keeping feature requests in discussions, and if it's not working out we're not opposed to moving them back to issues.
